app/javascript/tests/subject/MoveToHousehold.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { shallow } from 'enzyme';
+import { Button, Modal, Form } from 'react-bootstrap';
+import ReactTooltip from 'react-tooltip';
+import MoveToHousehold from '../../components/subject/MoveToHousehold.js'
+import { mockPatient1 } from '../mocks/mockPatients'
+
+const authyToken = "Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==";
+
+function getWrapper(patient) {
+    return shallow(<MoveToHousehold patient={patient} authenticity_token={authyToken} />);
+}
+
+function getPatientName(patient) {
```

Comment:
can we move this to the utils file for the component tests? there are other helper methods in there

Reply:
Yes, done here: fe7f862